src/journal/mmap-cache.h

```diff
@@ -26,24 +26,26 @@
 #define MMAP_CACHE_MAX_CONTEXTS 9
 
 typedef struct MMapCache MMapCache;
+typedef struct FileDescriptor FileDescriptor;
```

Comment:
Hmm, if we export this then the name might be a bit overly generic... Maybe rename this MMapFileDescriptor or so, just to clarify that this has to do with mmap management?

As soon as we exporting something beyond the most local scope I think we should take a bit of care with not using too generic names.